export const handleUnexpectedError = (err) => {
    console.error(err);
}